libr/bp/bp.c

```diff
@@ -97,7 +97,7 @@ R_API RBreakpointItem *r_bp_get_in(RBreakpoint *bp, ut64 addr, int rwx) {
 	r_list_foreach (bp->bps, iter, b) {
 		// eprintf ("---ataddr--- 0x%08"PFMT64x" %d %d %x\n", b->addr, b->size, b->recoil, b->rwx);
 		// Check addr within range and provided rwx matches (or null)
-		if (addr >= b->addr && addr <= (b->addr+b->size) && \
+		if (addr >= b->addr && addr < (b->addr+b->size) && \
```

Comment:
Its fixed in my commit after yours. Chk the diff

> On 15 May 2016, at 22:02, Joshua J. Drake <notifications@github.com> wrote:
> 
> In libr/bp/bp.c:
> 
> > @@ -97,7 +97,7 @@ R_API RBreakpointItem *r_bp_get_in(RBreakpoint *bp, ut64 addr, int rwx) {
> >  	r_list_foreach (bp->bps, iter, b) {
> >  		// eprintf ("---ataddr--- 0x%08"PFMT64x" %d %d %x\n", b->addr, b->size, b->recoil, b->rwx);
> >  		// Check addr within range and provided rwx matches (or null)
> > -		if (addr >= b->addr && addr <= (b->addr+b->size) && \
> > +		if (addr >= b->addr && addr < (b->addr+b->size) && \
> I took another shot but I'm not sure what you mean here.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>